### ISSUE TITLE
Missing word in config man page for input `events` setting

### ISSUE THREAD

**Issue by tdaniel22** [project-repo/cagebreak#25]
The configuration man page for the input send events setting is missing the `events` word:

```
diff --git man/cagebreak-config.5.md man/cagebreak-config.5.md
index 4eea4c5..c618314 100644
--- man/cagebreak-config.5.md
+++ man/cagebreak-config.5.md
@@ -123,7 +123,7 @@ by prepending a line with the # symbol.
        *dwt enabled|disabled*
                Enables or disables disable-while-typing for the specified input device.
 
-       *enabled|disabled|disabled_on_external_mouse*
+       *events enabled|disabled|disabled_on_external_mouse*
                Enables or disables send_events for specified input device. Disabling
                send_events disables the input device.
```

Slightly off topic: I was hoping to use this setting to hide the mouse cursor, but it doesn't work. My embedded system must expose a pointer device even though none is connected, altough I can only infer that much because the system is very bare bones and I can't check. Is there a way to do that?

**Comment by project-repo** [comment]
Since the word omission is fixed I have created #26 to discuss mouse hiding.

I am closing this issue.

cheers
project-repo

**Comment by project-repo** [comment]
This issue has been resolved in release 1.9.0.

cheers
project-repo
